src/components/VeggyHeader/HeaderSearchBar/HeaderSerchBar.jsx

```diff
@@ -1,28 +1,29 @@
-import './HeaderSearchBar.css'
-import arrow from '../../../../public/img/back.png';
+import './HeaderSearchBar.css';
+import backArrow from '../../../../public/img/back-arrow.png';
+
 export default function HeaderSearchBar (props) {
```

Comment:
де деструктуризація?
```suggestion
export default function HeaderSearchBar(props) {
```